registration/src/ndt.cpp

```diff
@@ -47,3 +48,5 @@
 template class PCL_EXPORTS pcl::NormalDistributionsTransform<pcl::PointXYZ, pcl::PointXYZ>;
 template class PCL_EXPORTS pcl::NormalDistributionsTransform<pcl::PointXYZI, pcl::PointXYZI>;
 template class PCL_EXPORTS pcl::NormalDistributionsTransform<pcl::PointXYZRGB, pcl::PointXYZRGB>;
+
+#endif
```

Comment:
Needs a '\n' at the last line to qualify as a POSIX text file

Reply:
Done